### ISSUE TITLE
CI constantly fails 

### ISSUE THREAD

**Issue by tomMisson** [tomMisson/ManMetHacks2.0-site#6]
CI will always fail due to content in the SVG's not being accessible to screen readers as a react warning.

**Comment by tomMisson** [comment]
-Aria-label was hidden
